ui/tests/acceptance/keyboard-test.js

```diff
@@ -344,6 +347,15 @@ module('Acceptance | keyboard', function (hooks) {
         'Shift+ArrowRight takes you to the next tab (Evaluations)'
       );
 
+      await triggerKeyEvent('.page-layout', 'keydown', 'ArrowRight', {
+        shiftKey: true,
+      });
+      assert.equal(
+        currentURL(),
+        `/jobs/${jobID}@default/clients`,
+        'Shift+ArrowRight takes you to the next tab (Clients)'
+      );
+
```

Comment:
Side-effect-of-a-side-effect:
- I wanted to make sure that keynav shift+right took you to the variables page, but you only see that if you have variable access
- so I said okay let's make the token a management token so that they'd definitely see it
- well the jobs created for this mock are all system jobs, which have client pages, which were otherwise unseen because before now the mocked token didn't have node read access, either, and now they do